certora/confs/ghost.conf

```
@@ -0,0 +1,23 @@
+#  A default configuration for a contract using ERC20s
```

```suggestion
//  A default configuration for a contract using ERC20s
```
All comments should start with a double slash